core/tracker/tracker.go

```diff
@@ -33,28 +33,33 @@ type component int
 
 // Core components arranged in the order data flows through them.
 const (
-	scheduler component = iota
+	zero component = iota
+	scheduler
 	fetcher
 	consensus
 	validatorAPI
 	parSigDBInternal
 	parSigEx
 	parSigDBThreshold
 	sigAgg
-
 	sentinel
 )
 
 // These constants are used for improving messages for why a duty failed.
 const (
-	fetcherMsg                  = "couldn't fetch duty data from the beacon node"
-	fetcherProposerThresholdMsg = "couldn't propose block due to insufficient partial randao signatures"
-	fetcherProposerMsg          = "couldn't propose block since randao duty failed"
-	consensusMsg                = "consensus algorithm didn't complete"
-	validatorAPIMsg             = "signed duty not submitted by local validator client"
-	parSigDBInternalMsg         = "partial signature database didn't trigger partial signature exchange"
-	parSigExMsg                 = "no partial signatures received from peers"
-	parSigDBThresholdMsg        = "insufficient partial signatures received, minimum required threshold not reached"
+	fetcherMsg                        = "couldn't fetch duty data from the beacon node"
+	fetcherAggregatorNoAttDataMsg     = "couldn't aggregate attestation due to failed attester duty"
+	fetcherAggregatorFewPreparesMsg   = "couldn't aggregate attestation due to insufficient partial committee subscriptions"
+	fetcherAggregatorZeroPreparesMsg  = "couldn't aggregate attestation due to zero partial committee subscriptions"
+	fetcherAggregatorFailedPrepareMsg = "couldn't aggregate attestation due to failed prepare aggregator duty "
+	fetcherProposerFewRandaosMsg      = "couldn't propose block due to insufficient partial randao signatures"
+	fetcherProposerZeroRandaosMsg     = "couldn't propose block due to zero partial randao signatures"
+	fetcherProposerFailedRandaoMsg    = "couldn't propose block due to failed randao duty"
+	consensusMsg                      = "consensus algorithm didn't complete"
+	validatorAPIMsg                   = "signed duty not submitted by local validator client"
+	parSigDBInternalMsg               = "partial signature database didn't trigger partial signature exchange"
+	parSigExMsg                       = "no partial signatures received from peers"
+	parSigDBThresholdMsg              = "insufficient partial signatures received, minimum required threshold not reached"
```

Comment:
@dB2510 @thomasheremans we should consider a collapsed section of the FAQ on the doc site that explains each of these errors in two sentences or so. We should aim that if people google these errors exactly our docs show up in the indexed google results.

Reply:
Sure @dB2510 can you provide me with the two sentences or so for each and I will add them to the docs

Reply:
Thanks @corverroos , @OisinKyne this has been added to the PR https://github.com/ObolNetwork/obol-docs/compare/main...Launchpad-guide-and-faq